peps/pep-0667.rst

```diff
@@ -3,13 +3,15 @@ Title: Consistent views of namespaces
 Author: Mark Shannon <mark@hotpy.org>,
         Tian Gao <gaogaotiantian@hotmail.com>
 Discussions-To: https://discuss.python.org/t/46631
-Status: Accepted
+Status: Final
 Type: Standards Track
 Created: 30-Jul-2021
 Python-Version: 3.13
 Post-History: 20-Aug-2021, 22-Feb-2024
 Resolution: https://discuss.python.org/t/pep-667-consistent-views-of-namespaces/46631/25
```

Comment:
While we're here: 
```suggestion
Resolution: `25-Apr-2024 <https://discuss.python.org/t/pep-667-consistent-views-of-namespaces/46631/25>`__
```

Reply:
Could also use: https://discuss.python.org/t/46631/25

Reply:
```suggestion
Resolution:  `25-Apr-2024 <https://discuss.python.org/t/46631/25>`__
```